iteration/iteration.go

```diff
@@ -26,19 +26,25 @@ const (
 	IterationStateClose    = "close"
 	PathSepInService       = "/"
 	PathSepInDatabase      = "."
+	IterationActive        = bool(true)
+	IterationNotActive     = bool(false)
 )
 
+var active bool
```

Comment:
You store the active state of a single iteration as a Global state?!

At this point it's random what status an iteration gets...